std_msgs/String type
AchievedGoal state_goal
sensor_msgs/Image img_goal
std_msgs/String lang_goal